src/test/java/seedu/address/logic/parser/ChargeCommandParserTest.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_CHARGE_AMY_INVALID_ARGS;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_CHARGE_AMY_INVALID_CHARGE_AMOUNT;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_CHARGE_AMY_INVALID_CHARGE_DATE;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_CHARGE_AMY_NO_CHARGE_AMOUNT;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_CHARGE_AMY_NO_CHARGE_DATE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CHARGEAMT_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CHARGEDATE_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CHARGE_AMY;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PET;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.ChargeCommand;
+import seedu.address.model.charge.Charge;
+import seedu.address.model.charge.ChargeDate;
+
+public class ChargeCommandParserTest {
+    private ChargeCommandParser parser = new ChargeCommandParser();
+
+    @Test
+    public void parse_allFieldsPresent_success() {
```

Comment:
Possible to add some javadocs here regarding each test case?

Reply:
Actually while I was writing test cases as well, I realised that there no java docs written for test cases by AB3. Is java docs required for test cases ? I dont think so.